CMakeLists.txt

```diff
@@ -214,6 +212,8 @@ if (NOT DEFINED Boost_VERSION_STRING)
     set(Boost_VERSION_STRING "${Boost_MAJOR_VERSION}.${Boost_MINOR_VERSION}.${Boost_SUBMINOR_VERSION}")
 endif()
 
+# date library: https://github.com/HowardHinnant/date/
+find_package(date REQUIRED)
```

Comment:
Why is this needed? Is this just to get an error if it is not installed?

Reply:
Yes it is just to get the error at cmake configuration rather than compilation.